hbase-server/src/main/java/org/apache/hadoop/hbase/master/LoadBalancer.java

```diff
@@ -86,6 +86,18 @@
   // We deliberately use 'localhost' so the operation will fail fast
   ServerName BOGUS_SERVER_NAME = ServerName.valueOf("localhost,1,1");
 
+  enum RunMode {
+    CHORE, REQUEST, FORCE, DRY_RUN;
```

Comment:
`FORCE` is only given by a user, right? (not something HBase does on its own)

What about calling this `FORCED_REQUEST` (or similar) instead to reflect that?